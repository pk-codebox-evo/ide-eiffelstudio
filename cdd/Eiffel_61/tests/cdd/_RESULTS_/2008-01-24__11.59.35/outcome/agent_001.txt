CDD_TEST_PROCEDURE_01.test_call
	Setup (exceptional)
		reason:  Postcondition violated.
		tag:     is_set_up

		class:   CDD_EXTRACTED_TEST_CASE
		feature: set_up

CDD_TEST_ROOT_CLASS_01.test_bar
	Setup (normal)
	Test (exceptional)
		reason:  Postcondition violated.
		tag:     false_postcondition

		class:   ROOT_CLASS
		feature: bar
	Teardown (normal)

CDD_TEST_ROOT_CLASS_02.test_make
	Setup (normal)
	Test (exceptional)
		reason:  Postcondition violated.
		tag:     false_postcondition

		class:   ROOT_CLASS
		feature: bar
	Teardown (normal)

