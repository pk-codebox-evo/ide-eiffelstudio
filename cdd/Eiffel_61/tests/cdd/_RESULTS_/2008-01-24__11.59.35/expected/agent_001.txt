No general support for agents yet
